### PR TITLE
dbcache: unexport cachedRepos.Repos

### PR DESCRIPTION
**Pull request by leolara**
I know this is a small change, but correct and my first contribution!

I talked with @keegancsmith and he said he could review it and see if it could be merged

## Test plan

Already tested, just refactoring from public to private

### CODE REVIEW

**Comment by leolara** [comment]
I have submitted the CLA

File: internal/database/dbcache/cached_indexable_repos.go [hunk 1]
```diff
@@ -78,12 +78,12 @@ func (s *IndexableReposLister) list(ctx context.Context, onlyPublic bool) (resul
...
 
 	cached, _ := cache.Load().(*cachedRepos)
-	repos, needsUpdate := cached.Repos()
+	repos, needsUpdate := cached.repos()
 	if !needsUpdate {
 		return repos, nil
 	}
 
-	// We don't have any repos yet, fetch them
+	// We don't have any epos yet, fetch them
```
Comment (keegancsmith) [inline internal/database/dbcache/cached_indexable_repos.go:86]:
```suggestion
	// We don't have any repos yet, fetch them
```

**Review by keegancsmith — APPROVED** [review]

**Comment by keegancsmith** [comment]
@cla-bot check

File: internal/database/dbcache/cached_indexable_repos.go (hunk 1 above)
Comment (leolara) [reply internal/database/dbcache/cached_indexable_repos.go]: Fixed

**Comment by leolara** [comment]
I signed the CLA again

**Comment by keegancsmith** [comment]
@cla-bot check

**Comment by keegancsmith** [comment]
@leolara Thanks!

**Pull request merged**